vulnerability-rating-taxonomy.json

```diff
@@ -380,67 +552,111 @@
               "name": "Strict-Transport-Security",
               "type": "variant",
-              "priority": 5
+              "priority": 5,
+              "mapping": {
+                "cwe": ["CWE-933"],
+                "cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+              }
             },
             {
               "id": "x_content_type_options",
               "name": "X-Content-Type-Options",
               "type": "variant",
-              "priority": 5
+              "priority": 5,
+              "mapping": {
+                "cwe": ["CWE-933"],
+                "cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+              }
             },
             {
               "id": "content_security_policy",
               "name": "Content-Security-Policy",
               "type": "variant",
-              "priority": 5
+              "priority": 5,
+              "mapping": {
+                "cwe": ["CWE-933"],
+                "cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+              }
             },
             {
               "id": "public_key_pins",
               "name": "Public-Key-Pins",
               "type": "variant",
-              "priority": 5
+              "priority": 5,
+              "mapping": {
+                "cwe": ["CWE-933"],
+                "cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+              }
             },
             {
               "id": "x_content_security_policy",
               "name": "X-Content-Security-Policy",
               "type": "variant",
-              "priority": 5
+              "priority": 5,
+              "mapping": {
+                "cwe": ["CWE-933"],
+                "cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+              }
             },
             {
               "id": "x_webkit_csp",
               "name": "X-Webkit-CSP",
               "type": "variant",
-              "priority": 5
+              "priority": 5,
+              "mapping": {
+                "cwe": ["CWE-933"],
+                "cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+              }
             },
             {
               "id": "content_security_policy_report_only",
               "name": "Content-Security-Policy-Report-Only",
               "type": "variant",
-              "priority": 5
+              "priority": 5,
+              "mapping": {
+                "cwe": ["CWE-933"],
+                "cvss_base_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N"
+              }
             },
             {
               "id": "cache_control_for_a_sensitive_page",
               "name": "Cache-Control for a Sensitive Page",
               "type": "variant",
-              "priority": 4
+              "priority": 4,
+              "mapping": {
+                "cwe": ["CWE-933", "CWE-524"],
+                "cvss_base_vector": "AV:N/AC:H/PR:N/UI:N/S:U/C:L/I:N/A:N"
```

Comment:
`CWE-524` should go first as a smaller number, this way we can ensure we are able to locate cwe fields with identical values. In this case there is cache_control_for_a_non_sensitive_page that has "cwe": ["CWE-524", "CWE-933"],